azkaban-common/src/test/java/azkaban/container/models/AzKubernetesV1PodBuilderTest.java

```diff
@@ -49,6 +49,8 @@ public void testPodSpec() throws IOException {
                 azConfVer)
             .addHostPathVolume("nscd-socket", "/var/run/nscd/socket", "Socket",
                 "/var/run/nscd/socket")
+            .addSecretVolume("azkaban-private-properties", "holdem6-azkaban-private-properties",
```

Comment:
Can you remove reference to Li-internal names, applies to yaml file as well?

Reply:
Good point. Fixed.